Resources/ProxyProvider/DIRACCAProxyProvider.py

```diff
@@ -51,11 +51,15 @@ def __init__(self, parameters=None):
         if nid not in self.n2field:
           self.n2field[nid] = field
         self.n2field[nid] = len(field) < len(self.n2field[nid]) and field or self.n2field[nid]
+    self.caDict = {}
```

Comment:
Does this still need to become a member variable? Or is it just left over from debugging?

It looks like it's only used by `getUserDN`.

Reply:
It's a part of future PR where I use self.caDict to setParameters as well.

Reply:
Then it should be part of that PR

Reply:
OK, I agree